wolfssl/wolfcrypt/port/atmel/atmel.h

```diff
@@ -93,10 +104,12 @@ enum atmelSlotType {
     ATMEL_SLOT_DEVICE,
     ATMEL_SLOT_ECDHE,
     ATMEL_SLOT_ECDHE_ENC,
+    ATMEL_SLOT_ECDHE_ALICE,
+    ATMEL_SLOT_ECDHE_BOB,
 };
 
-int  atmel_ecc_alloc(int slotType);
-void atmel_ecc_free(int slotId);
+WOLFSSL_API int  atmel_ecc_alloc(int slotType);
+WOLFSSL_API void atmel_ecc_free(int slotId);
```

Comment:
public API should have wolfSSL_* or wolfCrypt_* names

Reply:
not resolved yet. Public API need our naming scheme prepended to them. Preferably matching the same style as other public API in this header file.